core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -525,7 +523,7 @@ protected boolean runSerially() {
                 // process and drain cmd queue
                 useful = true;
             }
-            if (last + checkInterval < t) {
+            if (last + configuration.getIdleCheckInterval() < t) {
```

Comment:
Better to not copy the value for lifetime but get it from config for testing and if we will want to ever make configuration dynamic

Reply:
this is premature, when we come to define dynamic configuration we will be revisiting values one by one. This change doesn't solve the issue of dynamic configuration but potentially adds a gotcha should latency of `getIdleCheckInterval()` increase

Reply:
I think we can accept this "potential" latency given it's not in a tight loop and not on hot path for sake of mocking value in the tests, cannot we?

Reply:
ok, I reverted this change